app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -328,7 +328,7 @@ def bulk_scope
         scope = resource_scope
         scope = scope.search_for(params[:search]) if params[:search]
         scope = scope.select('DISTINCT foreman_tasks_tasks.*')
-        scope.where(:id => params[:task_ids]) if params[:task_ids]
+        scope = scope.where(:id => params[:task_ids]) if params[:task_ids]
```

Comment:
Can you use strong parameters?

Reply:
Can you please explain for what? :)

Reply:
because of the `controller` in only noticed now that it is actually the API file,
@adamruzicka, can we also use them in API? like in the first example [here](https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html) for controllers

Reply:
As far as I know nothing prevents us from using them here.

Reply:
I'm not sure where is "here"? as this line doesn't assign anything

Reply:
"Here" was meant as "in this controller". Strong parameters are a way of expressing which parameters are required and which are allowed in the request params

Reply:
Thanks for explaining! It's out of the scope of this PR and maybe @laviro can you open an issue with an explanation?

Reply:
it would reduce all of those params checking,
no probs, opened: https://projects.theforeman.org/issues/29547